compiler/luci/service/src/Nodes/CircleStridedSlice.cpp

```diff
@@ -311,6 +311,8 @@ StridedSliceParams BuildStridedSliceParams(StridedSliceContext *op_context)
   // Calculate effective_input_shape and its corresponding begin, end, strides.
   loco::TensorShape input_shape = circle_shape(op_context->input);
   int64_t added_ellipsis = 0, added_axises = 0;
+  // make sure no overflow
+  assert(static_cast<int32_t>(effective_dims) == effective_dims);
   op_context->effective_input_shape.rank(effective_dims);
```

Comment:
AFAIK, `rank` is uint32_t.

```suggestion
  // make sure no overflow
  assert(static_cast<uint32_t>(effective_dims) == effective_dims);
  op_context->effective_input_shape.rank(effective_dims);
```